resources/views/admin/print/free.blade.php

```diff
@@ -20,12 +20,14 @@
                 required onfocus="M.Datepicker.getInstance(deadline).open();">
             <label for="deadline">@lang('print.deadline')</label>
             <script>
+            var tomorrow = new Date();
+            tomorrow.setDate(new Date().getDate()+1);
             $(document).ready(function() {
                 $('.datepicker').datepicker({
                     format: 'yyyy-mm-dd',
                     firstDay: 1,
                     yearRange: 10,
-                    minDate: today,
+                    minDate: tomorrow,
```

Comment:
To match the validation rules.

Reply:
We might want to consider a more fine grained deadline, but it might be an overkill, I think it's okay.
Just to be clear, when the date is reached, the pages won't be available, or they will work on that day too?

Reply:
I think it will work on the last day, but we might test it :D
Maybe we can find a bug also

Reply:
I am writing the tests. Once #222 is in, I'll send in my PR.